src/types.rs

```diff
@@ -1519,7 +1722,27 @@ impl Rule {
             None
         }
     }
-    /// Check whether the rule contains certain clauses.
+    /// Check whether the `Rule` contains certain clauses.
+    ///
+    /// # Examples
+    /// 
+    /// ```
+    /// # use term_rewriting::{Signature, Term, Rule, parse_rule};
+    /// # use std::collections::HashMap;
+    /// let mut sig = Signature::default();
+    ///
+    /// let mut r = parse_rule(&mut sig, "A(x_) = B | C").expect("parse of A = B | C");
+    /// let mut r2 = parse_rule(&mut sig, "A(y_) = B").expect("parse of A = B");
```

Comment:
Looks like your `expect` strings are slightly out of sync with the `parse_rule` strings.